### DISCUSSION TITLE
Add the following commit from A1111 a1219e84e2a00d1f75613e92e6b0a70ff94d03f2 "hires button, opt insert gallery"

### DISCUSSION THREAD

**Discussion by nailz420** [lllyasviel/stable-diffusion-webui-forge#2323] [Ideas]
Please add the following commit [hires button, opt insert gallery](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/a1219e84e2a00d1f75613e92e6b0a70ff94d03f2)

Thanks,
Tim

**Comment by andy8992** [comment]
sounds nice
